Tests/LibWeb/AccessibilityBridge/tests_orca/test_focus_changed.py

```
@@ -0,0 +1,88 @@
+        """Drive _on_focused_changed via the synthetic event. Asserts no AttributeError."""
+        event = self._build_focused_event(source)
+        try:
+            self.orca.script._on_focused_changed(event)
+        except AttributeError as exc:
+            self.fail(
+                f"_on_focused_changed raised AttributeError — likely a pre-Orca-50 API call slipping back in: {exc}"
+            )
+        except Exception:
+            # Other exceptions (TypeError from a partially-constructed event, errors deep in super's call chain
+            # against the synthetic environment, etc.) aren't what this regression test is guarding against.
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)